diff --git a/usr/lib/opengl/nvidia/xorg.conf b/usr/lib/opengl/nvidia/xorg.conf
index 733250c..7fca1e7 100644
--- a/usr/lib/opengl/nvidia/xorg.conf
+++ b/usr/lib/opengl/nvidia/xorg.conf
@@ -39,6 +39,7 @@ EndSection
 Section "Device"
     Identifier     "Device0"
     Driver         "nvidia"
+    Option         "RegistryDwords" "EnableBrightnessControl=1"
     VendorName     "NVIDIA Corporation"
 EndSection
 
